tests/test_prometheus.py

```
@@ -0,0 +1,127 @@
+    def test_default_binds_localhost(self):
+        with mock.patch.object(
+            prometheus_client, 'start_http_server'
+        ) as start:
+            prometheus.start(9123)
+        start.assert_called_once_with(9123, addr='127.0.0.1')
+
+    def test_explicit_address(self):
+        with mock.patch.object(
+            prometheus_client, 'start_http_server'
+        ) as start:
+            prometheus.start(9123, address='0.0.0.0')
```

Ruff (0.15.20)

[error] Possible binding to all interfaces

(S104)